AndelaOSP/art-dashboard
# [#159556607] Validate Assign Asset

## What does this PR do?
Disable `Assign Asset` button when no user is assigned to asset. This is to prevent the user from clicking `Assign Asset` without filling in a user.
## Description of Task to be completed?
- add button state to `Assign Asset` button
- change state from disabled when user selected in dropdown
- change css for button and refactor css to allow for reuse.
## How should this be manually tested?
`npm test` OR
- Login to ART
- Go to the assets page
- click on asset (one that is unassigned)
- you will see the `Assign Asset` button is disabled
- when selecting a user, the `Assign Asset` becomes clickable
## What are the relevant pivotal tracker stories?
[159556607](https://www.pivotaltracker.com/story/show/159556607)
## Any background context you want to add?
n/a
## Important notes
n/a
## Packages installed
n/a
## Deployment note
n/a
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
<img width="1177" alt="screen shot 2018-08-13 at 17 21 13" src="https://user-images.githubusercontent.com/6042152/44037881-210d2456-9f1e-11e8-9038-2257dcd2acec.png">
